src/dev/build/tasks/bundle_fleet_packages.ts

```diff
@@ -40,14 +43,38 @@ export const BundleFleetPackages: Task = {
 
     log.debug(
       `Found configured bundled packages: ${parsedFleetPackages
-        .map((fleetPackage) => `${fleetPackage.name}-${fleetPackage.version}`)
+        .map((fleetPackage) => `${fleetPackage.name}-${fleetPackage.version || 'latest'}`)
         .join(', ')}`
     );
 
     await Promise.all(
       parsedFleetPackages.map(async (fleetPackage) => {
-        const archivePath = `${fleetPackage.name}-${fleetPackage.version}.zip`;
-        const archiveUrl = `${eprUrl}/epr/${fleetPackage.name}/${fleetPackage.name}-${fleetPackage.version}.zip`;
+        const stackVersion = config.getBuildVersion();
+
+        let versionToWrite = fleetPackage.version;
+        let versionToDownload = versionToWrite;
+
+        // Resolve the latest version (including prerelease version) from EPR if
+        // the bundled package record opts into the `forceAlignStackVersion` behavior
+        if (fleetPackage.forceAlignStackVersion) {
+          const searchUrl = `${eprUrl}/search?package=${fleetPackage.name}&experimental=true`;
```

Comment:
Should this include `kibana.version` query parameter? i.e. to don't get the latest version, but the latest compatible version?

Reply:
@axw Good callout. Based on our discussion in the parent issue, though, I've removed the "find latest version of the package" logic here. We expect a version to be defined in `fleet-package.json` now, and Kibana will take care of rewriting that version when bundling the package.

Reply:
Perfect, thanks!